shimmingtoolbox/cli/referencemaps.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import logging
+
+import click
+
+from .. import referencemaps
+
+@click.command()
+@click.option("--verbose", is_flag=True, help="Be more verbose.")
+@click.argument("inputs", required=True, nargs=-1, type=click.File('rb'))
+@click.argument("output", type=click.Path(exists=False))
+def main(verbose, inputs, output):
+    """
+    Compute a reference field b0 fieldmap OUTPUT from a folder of dicom files INPUT.
+    """
+    if verbose:
+        logging.getLogger().setLevel(logging.INFO)
```

Comment:
should we decide the logger levels to use since there's a lot. ERROR and INFO are straight forward but like what would be a CRITICAL or a WARNING?

Reply:
My thinking about logging is that Golang is right: [log levels are unhelpful](https://dave.cheney.net/2015/11/05/lets-talk-about-logging).

This gives three levels of output: print()s, to show the normal course of output, verbose logging, to show details that might help a user debug why it's not working for them, and exceptions, which immediately exit with an error. Any more is unhelpfully splitting hairs, I think.